Formula/gecode.rb

```diff
@@ -12,8 +12,21 @@ class Gecode < Formula
     sha256 "a6cf500df618c42f0668bb227c090e6c1d3d3369c8b4537220d3deb78d5f8286" => :mountain_lion
   end
 
+  depends_on "qt5" => :optional
+
   def install
-    system "./configure", "--prefix=#{prefix}", "--disable-examples"
+    args = %W[
+      --prefix=#{prefix}
+      --disable-examples
+    ]
+    ENV.append "CXXFLAGS", "-std=c++11"
+    if build.with? "qt5"
+      args << "--enable-qt"
+      ENV.append_path "PKG_CONFIG_PATH", "#{HOMEBREW_PREFIX}/opt/qt5/lib/pkgconfig"
+    else
+      args << "--disable-qt"
+    end
+    system "./configure", *args
     system "make", "install"
   end
 end
```

Comment:
Any chance you could add a simple test e.g. ideally something more substantial than e.g. `--version` or `--help` but just use them if necesssary? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!